hnn_core/viz.py

```diff
@@ -714,17 +737,25 @@ def plot_cell_connectivity(net, conn_idx, src_gid, colorbar=True,
     src_idx = np.where(src_range == src_gid)[0][0]
     src_pos = src_type_pos[src_idx]
 
-    weights, x_pos, y_pos = list(), list(), list()
+    # Aggregate positions and weight of each connected target
+    weights, target_x_pos, target_y_pos = list(), list(), list()
     for target_idx in target_indeces:
         target_pos = target_type_pos[target_idx]
-        x_pos.append(target_pos[0])
-        y_pos.append(target_pos[1])
+        target_x_pos.append(target_pos[0])
+        target_y_pos.append(target_pos[1])
         weight, _ = _get_gaussian_connection(src_pos, target_pos, nc_dict)
         weights.append(weight)
 
-    im = ax.scatter(x_pos, y_pos, c=weights)
+    im = ax.scatter(target_x_pos, target_y_pos, c=weights, cmap=colormap)
+
+    # Gather positions of all gids in target_type.
+    x_pos = [target_type_pos[idx][0] for idx in range(len(target_type_pos))]
+    y_pos = [target_type_pos[idx][1] for idx in range(len(target_type_pos))]
+    ax.scatter(x_pos, y_pos, color='k', zorder=-1, s=4)
```

Comment:
just a thought, should we make this a little greyish to that people are not confused that the size stands for something ... it was not apparent to me because the red dot was different size, then there were the black dots and then there was the connectivitiy dots all different sizes. Maybe you can also think of changing the symbol of the `src_gid` dot from circle to square or something. Sorry, I know this is a bit bikeshedding at this point ...

Reply:
Good suggestions. I think to be more explicit, I'll do a red square for `src_gid`, and black `x`'s for the non-connected cells.